src/testing/jenkins/Fossology_Source_Install.xml

```diff
@@ -100,19 +100,20 @@ else
   fi
 fi
 #svn info;
-#echo http-proxy-host = web-proxy.cce.hp.com &gt;&gt; ~/.subversion/servers;
+#echo http-proxy-host = xxx &gt;&gt; ~/.subversion/servers;
 #echo http-proxy-port = 8088 &gt;&gt; ~/.subversion/servers;
 #svn co https://svn.code.sf.net/p/fossology/code/trunk/fossology;
-git config --global http.proxy &quot;http://web-proxy.cce.hp.com:8088&quot;;
+git config --global http.proxy &quot;http://xxx:8088&quot;;
 #git config pack.windowMemory 10m;
 #git config pack.packSizeLimit 20m;
-git config --global https.proxy &quot;http://web-proxy.cce.hp.com:8088&quot;;
+git config --global https.proxy &quot;http://xxx:8088&quot;;
 git clone --depth 1 https://github.com/fossology/fossology.git fossology;
 pwd;
-export http_proxy=http://web-proxy.cce.hp.com:8088/
+export http_proxy=http://xxx:8088/
 cd ./fossology/;
 ./utils/fo-installdeps -e -y;
-curl -sS --proxy http://web-proxy.cce.hp.com:8088 https://getcomposer.org/installer | php &amp;&amp; mv composer.phar /usr/local/bin/composer;
+make composer_download
+make composer_install
```

Comment:
The `composer_install` target is also called when `make install` is called, i.e. two lines below (see: https://github.com/fossology/fossology/blob/ae754b26e849b4c9068ed6f4aa354310165aec0c/src/Makefile#L53)